paasta_tools/cli/cmds/spark_run.py

```diff
@@ -188,6 +191,14 @@ def add_subparser(subparsers):
         ),
         default=None,
     )
+    list_parser.add_argument(
+        "--docker-disk-limit",
+        help=(
+            "Set docker disk limit."
+            "Note: The default disk limit will be same as the memory limit if this argument is not provided."
```

Comment:
just checking: could we move the default any lower? tron-used batch boxes don't really have particularly large disks (i just hopped onto a random one and there's only ~160GB of disk free) and the human-used batch boxes have enough folks running spark on them that the lower we can make the default, the better

Reply:
@88manpreet Could you please have a look at this?